api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -7,10 +7,355 @@ type: Object
+
+
+    ##### Vue
+
+        <!-- tab: App.vue -->
+        <template>
+            <DxDataGrid ...
+                @exporting="onExporting">
+                <DxExport
+                    :enabled="true"
+                />
+            </DxDataGrid>
+        </template>
+
+        <script>
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import { DxDataGrid, DxExport } from 'devextreme-vue/data-grid';
+        import { exportDataGrid } from 'devextreme/excel_exporter';
+        import ExcelJS from 'exceljs';
+        import saveAs from 'file-saver';
+
+        export default {
+            components: {
+                DxDataGrid,
+                DxExport
+            },
+            methods: {
+                onExporting(e) {
+                    const workbook = new ExcelJS.Workbook();
+                    const worksheet = workbook.addWorksheet('Main sheet');
+                    exportDataGrid({
+                        component: e.component,
+                        worksheet: worksheet
+                    }).then(function() {
+                        workbook.xlsx.writeBuffer()
+                            .then(function(buffer) {
+                                saveAs(new Blob([buffer], { type: 'application/octet-stream' }), 'DataGrid.xlsx');
+                            });
+                    });
+                    e.cancel = true;
+                }
+            }
+        }
+        </script>
+
+    ##### React
+
+        <!-- tab: App.js -->
+        import React from 'react';
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import DataGrid, { Export } from 'devextreme-react/data-grid';
+        import ExcelJS from 'exceljs';
+        import saveAs from 'file-saver';
+        import { exportDataGrid } from 'devextreme/excel_exporter';
+
+        class App extends React.Component {
+            render() {
+                return (
+                    <DataGrid ...
+                        onExporting={this.onExporting}>
+                        <Export enabled={true} />
+                    </DataGrid>
+                );
+            }
+            onExporting(e) {
+                const workbook = new ExcelJS.Workbook();
+                const worksheet = workbook.addWorksheet('Main sheet');
+                exportDataGrid({
+                    component: e.component,
+                    worksheet: worksheet
+                }).then(function() {
+                    workbook.xlsx.writeBuffer()
+                        .then(function(buffer) {
+                            saveAs(new Blob([buffer], { type: 'application/octet-stream' }), 'DataGrid.xlsx');
+                        });
+                });
+                e.cancel = true;
+            }
+        }
+        export default App;
+
+    --- 
+
+4. **Further Configure Export and Customize Data**      
```

Comment:
The entire step 4 can now be removed. The demo links are alright without any preceding text.